### DISCUSSION TITLE
how to fix render bug with vmware

### DISCUSSION THREAD

**Discussion by bravedutch** [sqlitebrowser/sqlitebrowser#3235] [Q&A]
When run a program in vmware, part of the screen is rendered black. How do I solve it?

**Comment by justinclift** [comment]
That sounds more like a VMware rendering bug than anything else.

That being said, which host platform are you using, which VMware product, and which guest OS is it happening in?

**Reply by bravedutch** [reply]
I running Ubuntu 22 with VM Workstation Pro 17 on Win 11.

**Reply by justinclift** [reply]
k.  Just to make sure I'm reading that right, you're meaning:

* The host is running Win 11
* The guest is Ubuntu 22.<something>

Hmmm, nothing is standing out as super wrong with that.  I'd check two things first:

1. Make sure your Ubuntu guest is running the `open-vm-tools` package (eg `sudo apt install open-vm-tools`)
2. Take a look at the "Display" settings in VMware for the VM.  Check if the "Accelerate 3D graphics" open is enabled, and check how much graphics memory is assigned to guests.  Playing around with those settings _might_ change the behaviour in a useful way.

Oh, on that note, what type of graphics card or graphics adapter is your host system using?  It _might_ be useful to know, for doing searches online (Google, etc) when looking for solutions. :smile:

And a screenshot of the problem is probably a good idea too. :wink:
